src/components/SignUp.jsx

```diff
@@ -0,0 +1,96 @@
+import {React, useState} from 'react';
+import {Container, Row, Form, Button} from 'react-bootstrap';
+import {useNavigate} from 'react-router-dom';
+
+
+async function signUp(user) {
+  const baseUrl = process.env.API_BASE_URL || 'http://localhost:8000';
+  return fetch(`${baseUrl}/api/users/signUp`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+    body: JSON.stringify(user),
+  })
+      .then((data) => data.json())
+      .catch((err)=>console.log(err));
+}
+
+
+const SignUp = () => {
+  const navigate = useNavigate();
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [rePassword, setRePassword] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const response = await signUp({username, email, password, rePassword});
+    console.log(response);
```

Comment:
remove

Reply:
leaving for now for testing later

Reply:
see comments above. This is a PR to master. You don't want to commit the random logs you used for testing. You can always add them back later on your local.